general_rules/ExternalComputing.tex

```diff
@@ -26,6 +26,34 @@ \section{External computing}\label{rule:robot_external_computing}
 
 \textbf{Remark:} Teams are allowed to use their own software in the external computing devices (not only cloud services). This software must be publicly available to other teams for scientific purposes (evaluation, test, and benchmarking), as well as for TC for inspection. Although open-sourcing the software is not mandatory, this practice is advised and encouraged by the league.
 
+\subsection{External Computing Devices on site for SSPL (Hardware)}
+\label{rule:robot_external_computing_devices_sspl}
+
+\begin{itemize}
+  \item \textbf{Location:} all external computing devices, approved by the \iaterm{Technical Committee}{TC}
+  		(see~\ref{rule:robot_external_computing}), must be placed at a designated location close to the arena.
+  		The location features a table (or otherwise suitable furnishing) and an ethernet switch (wired)
+        that connects to the arena WIFI. In the remainder of this document, this location is called
+  		\textbf{E}xternal \textbf{C}omputing \textbf{R}esource \textbf{A}rea (ECRA).
+  \item \textbf{Procedure:} before a teams' test run, external comp. devices must be setup in time.
```

Comment:
I find the paragraph a little bit long. We must also avoid abbreviations like *comp.*. My two cents:
*There may be up to \textbf{two} teams in ECRA at any time, the testing team and the upcoming one. Immediately after the test, the team must remove all equipment from ECRA.*

Reply:
Two teams is too little IMHO, let's make it three.